instances/widgets.treasury-factory.near/widget/pages/dashboard/Portfolio.jsx

```diff
@@ -352,7 +352,8 @@ const isLoading =
   ftTokens === null ||
   nearBalances === null ||
   nearPrice === null ||
-  !nearBalances.total;
+  !nearBalances.total ||
+  (isLockupContract && !lockupStartDate);
```

Comment:
what is this change doing?

Reply:
Ori saw this, so to avoid such null condition, we wait for the startDate and other parameter to set, until then we show the loader.
![image](https://github.com/user-attachments/assets/6dff19da-605a-41fa-ab3f-62c15e961324)